crates/cansat-stm32f4/src/tasks.rs

```diff
@@ -85,6 +93,23 @@ fn read_measurements(ctx: &mut app::idle::Context) -> Measurements {
     data
 }
 
+fn send_lora_package(lora: &mut crate::Lora) {
+    let mut resp_buffer: [u8; 255] = [0; 255];
+
+    match lora.send(b"AT+TEST=TXLRSTR,\"TEST_MSG\"\r\n", &mut resp_buffer) {
+        Ok(resp_len) => {
+            if let Err(e) = parse::response(&resp_buffer) {
+                defmt::error!("Lora error reponse: {}", defmt::Debug2Format(&e));
+            } else {
+                defmt::error!("Lora package sent {}", resp_len);
+            }
+        }
+        Err(e) => {
+            defmt::error!("Could not send lora package: {}", defmt::Debug2Format(&e));
+        }
+    }
+}
```

Comment:
That's a bit too much conditionals just to log something. Propagate the error and log it at a call site, you can `match` over the variants. Remove Debug2Format, once the Format is implemented.